_data/versions.yml

```diff
@@ -1,3 +1,7 @@
+versions:
+  currentReleaseStream: v2.5
```

Comment:
Does this need to be a sub-field of versions? Could it be its own top-level field?

```yaml
currentReleaseStream: v2.5
versions:
  ...
```